lib/ClangImporter/ImporterImpl.h

```diff
@@ -490,6 +490,11 @@ class LLVM_LIBRARY_VISIBILITY ClangImporter::Implementation
   /// pairs.
   llvm::DenseMap<std::pair<FuncDecl *, FuncDecl *>, SubscriptDecl *> Subscripts;
 
+  /// Keep track of getter/setter pairs for functions
+  /// imported from C++ subscript operators
+  /// based on the type of their parameter.
+  llvm::DenseMap<Type, std::pair<FuncDecl *, FuncDecl *>> CXXSubscripts;
```

Comment:
1. Can you reflow this comment. It looks like it's not taking up the full 80 lines.
2. Please say in the comment how first and second correspond to get and set.

Reply:
Done